topic_modeling/scrape_training_data/get_article_links.py

```diff
@@ -5,8 +5,11 @@
 from IPython.core.display import clear_output
 from time import time
 
-pages = [str(page_counter) for page_counter in range(0, 4)]  # change range to 0,50 when using cluster
-year_urls = [years_counter for years_counter in range(2015, 2016)]  # change range to 2012,2019 when using cluster
+# change range to 0,50 when using cluster
+pages = [str(page_counter) for page_counter in range(0, 4)]
+
+# change range to 2012,2019 when using cluster
+year_urls = [years_counter for years_counter in range(2015, 2016)]
```

Comment:
```suggestion
year_urls = list(range(2015, 2016))
```
?

Reply:
oh, duh. that seems very obvious is retrospect. i was very focused on "submitting" something for hw!

Reply:
fixed minor comments here: https://github.com/emtempleton/topic_modeling/pull/6/commits/e2b78762027a209f6fe64adb2854674e91e128bc